src/index.js

```diff
@@ -103,10 +111,14 @@ function mergeExtensionConfig(targetVersion, mergeTarget, extensionConfig, modul
   return deepmerge(mergeTarget, toMerge);
 }
 
-/** @type {Object|boolean} */
-const hyvaThemesConfig = basePath
-  ? JSON.parse(fs.readFileSync(path.join(basePath, hyvaThemeJsonInModule)))
-  : false;
+const hyvaThemesConfig = (() => {
+  try {
+    return basePath ? JSON.parse(fs.readFileSync(path.join(basePath, hyvaThemeJsonInModule))) : {};
```

Comment:
Previously the exported value was `undefined` if there was no config file. Now it would be an empty array.
This is a backward incompatible change. Please revert.

Reply:
✅